src/app/(views)/(public)/a/[archiveId]/loading.tsx

```diff
@@ -1,9 +1,8 @@
 import Spinner from "~/components/custom/spinner";
```

Comment:
[Blocking] Remove `Spinner` import as this is no longer required

Reply:
Okie